### PR TITLE
use testing app to read logfiles remotely in acceptance tests

### PR DESCRIPTION
**Pull request by individual-it**
## Description
acceptance test steps that work with the log-file can now do their job through the testing app https://github.com/owncloud/testing/pull/9

## Related Issue
- Fixes https://github.com/owncloud/QA/issues/590

## Motivation and Context
test remote systems

## How Has This Been Tested?
- run firewall acceptance tests that use this steps
- run api acceptance tests that use this steps

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [x] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/bootstrap/Logging.php [hunk 1]
```diff
@@ -111,7 +114,11 @@ public function theLastLinesOfTheLogFileShouldContainEntriesWithTheseAttributes(
 	public function theLogFileShouldNotContainAnyLogEntriesWithTheseAttributes(
 		$withOrContaining, TableNode $logEntriesExpectedNotToExist
 	) {
-		$logLines = \file(LoggingHelper::getLogFilePath());
+		$logLines = LoggingHelper::getLogFileContent(
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/Logging.php]: Note: when testing on some "independent" remote system, the existing log file might be large (many MB). It depends when the sysadmin of that system bothers to clear it. (That is a problem for some future time, if we start black-box testing, e.g. demo, systems that are up and running and in general use...)

File: tests/acceptance/features/bootstrap/Logging.php [hunk 2]
```diff
@@ -191,7 +198,11 @@ public function owncloudLogTimezoneIsSetTo($timezone) {
 	 * @throws \Exception
 	 */
 	public function theOwncloudLogIsCleared() {
-		LoggingHelper::clearLogFile();
+		LoggingHelper::clearLogFile(
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/Logging.php]: Note: when testing someone's remote system, we are going to sometimes clear their log file. But we are going to create ``user``, ``grp1`` on their system also, during testing. I guess this whole "black-box testing" thing is not supposed to be a toll for checking real production systems!

**Review by phil-davis — APPROVED** [review]
Looks good, and re-running CI with the updated testing app should pass.

**Comment by individual-it** [comment]
backport in #32536

**Pull request merged**